src/Microsoft.AspNet.Mvc.Core/ConsumesAttribute.cs

```diff
@@ -56,10 +56,10 @@ public void OnResourceExecuting(ResourceExecutingContext context)
                 MediaTypeHeaderValue requestContentType = null;
                 MediaTypeHeaderValue.TryParse(context.HttpContext.Request.ContentType, out requestContentType);
 
-                // Only execute if this is the last filter before calling the action.
```

Comment:
Does this comment go with another block in this method? Your new comment is great, but methinks the previous comment wandered away from home.